docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst

```diff
@@ -474,6 +474,20 @@ If you want to see if the File daemon has properly opened the port and is listen
 
    netstat -an | findstr 910[123]
 
+Windows Antivirus
+-----------------
+
+.. index:: Antivirus; Windows
+.. index:: Windows; Antivirus
+
+The Windows builtin anti-virus **Defender** is enabled by default. If you care about performance, it is advised to add the :command:`bareos-dir.exe`, :command:`bareos-sd.exe`, :command:`bareos-fd.exe` to the list of excluded monitored processes. Otherwise you may suffer by i/o throughput and excessive cpu consumption.
+
+.. image:: /include/images/windows-defender-exclusion.png
```

Comment:
The picture is very blurry. Maybe that can be improved?

Reply:
I will retry another one. seems compression and resize of this one destroyed the quality

Reply:
Looks much better now!